### PR TITLE
feat(plasma-tokens): auto generate SBOL theme

### PR DESCRIPTION
**Pull request by gibushnev**
:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.513.5021820057.swift)  
[color_sberHealth_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.513.5021820057.kt)  
[color_sberHealth_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.513.5021820057.ts)  
[color_sbermarket_business_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.513.5021820057.swift)  
[color_sbermarket_business_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.513.5021820057.kt)  
[color_sbermarket_business_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.513.5021820057.ts)  
[color_sbermarket_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.513.5021820057.swift)  
[color_sbermarket_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.513.5021820057.kt)  
[color_sbermarket_metro_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.513.5021820057.swift)  
[color_sbermarket_metro_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.513.5021820057.kt)  
[color_sbermarket_metro_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.513.5021820057.ts)  
[color_sbermarket_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.513.5021820057.ts)  
[color_sbermarket_selgros_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.513.5021820057.swift)  
[color_sbermarket_selgros_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.513.5021820057.kt)  
[color_sbermarket_selgros_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.513.5021820057.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.513.5021820057.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.513.5021820057.kt)  
[color_sbermarket_wlbusiness_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.513.5021820057.ts)  
[color_sberonline_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.513.5021820057.swift)  
[color_sberonline_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.513.5021820057.kt)  
[color_sberonline_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.513.5021820057.ts)  
[color_sberprime_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.513.5021820057.swift)  
[color_sberprime_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.513.5021820057.kt)  
[color_sberprime_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.513.5021820057.ts)  
[shadow_sbermarket_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.513.5021820057.ts)  
[typo_mage_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.513.5021820057.swift)  
[typo_mage_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.513.5021820057.kt)  
[typo_mage_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.513.5021820057.ts)  
[typo_plasma_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.513.5021820057.swift)  
[typo_plasma_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.513.5021820057.kt)  
[typo_plasma_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.513.5021820057.ts)  
[typo_ruler_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.513.5021820057.swift)  
[typo_ruler_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.513.5021820057.kt)  
[typo_ruler_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.513.5021820057.ts)  
[typo_sage_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.513.5021820057.swift)  
[typo_sage_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.513.5021820057.kt)  
[typo_sage_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.513.5021820057.ts)  
[typo_sbermarket_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.513.5021820057.swift)  
[typo_sbermarket_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.513.5021820057.kt)  
[typo_sbermarket_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.513.5021820057.ts)  
[typo_soulmate_ios-swift--canary.513.5021820057.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.513.5021820057.swift)  
[typo_soulmate_kotlin--canary.513.5021820057.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.513.5021820057.kt)  
[typo_soulmate_react-native--canary.513.5021820057.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.513.5021820057.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.212.0-canary.513.5021820057.0
  npm install @salutejs/plasma-core@1.117.0-canary.513.5021820057.0
  npm install @salutejs/plasma-hope@1.212.0-canary.513.5021820057.0
  npm install @salutejs/plasma-icons@1.146.0-canary.513.5021820057.0
  npm install @salutejs/plasma-temple@1.161.0-canary.513.5021820057.0
  npm install @salutejs/plasma-tokens@1.52.0-canary.513.5021820057.0
  npm install @salutejs/plasma-ui@1.193.0-canary.513.5021820057.0
  npm install @salutejs/plasma-web@1.212.0-canary.513.5021820057.0
  npm install @salutejs/plasma-cy-utils@0.59.0-canary.513.5021820057.0
  npm install @salutejs/plasma-sb-utils@0.115.0-canary.513.5021820057.0
  # or 
  yarn add @salutejs/plasma-b2c@1.212.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-core@1.117.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-hope@1.212.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-icons@1.146.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-temple@1.161.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-tokens@1.52.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-ui@1.193.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-web@1.212.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-cy-utils@0.59.0-canary.513.5021820057.0
  yarn add @salutejs/plasma-sb-utils@0.115.0-canary.513.5021820057.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-513/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-513/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-513/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-513/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-513/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-513/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-513/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-513/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-513/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

Top 5 significant render changes:
packages/plasma-ui/src/components/Pickers/Pickers.perftest.tsx#DatePicker: -18.72%

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-513/

**Review by neretin-trike — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-513/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-513/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-513/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-513/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-513/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-tokens/src/themes/sberonline__light.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        '--inverseSurfaceTransparentDeep': '#FFFFFFA3',
+        '--backgroundPrimary': '#F9F9F9FF',
+        '--darkBackgroundPrimary': '#080808FF',
+        '--lightBackgroundPrimary': '#F9F9F9FF',
+        '--inverseBackgroundPrimary': '#080808FF',
+        '--skeletonGradient':
+            'linear-gradient( 90deg, rgba(8, 8, 8, 0.09) 0%, rgba(8, 8, 8, 0.08) 6.25%, rgba(8, 8, 8, 0.05) 12.5%, rgba(8, 8, 8, 0.01) 25%, rgba(8, 8, 8, 0.05) 37.5%, rgba(8, 8, 8, 0.08) 43.75%, rgba(8, 8, 8, 0.09) 50%, rgba(8, 8, 8, 0.08) 56.25%, rgba(8, 8, 8, 0.05) 62.5%, rgba(8, 8, 8, 0.01) 75%, rgba(8, 8, 8, 0.05) 87.5%, rgba(8, 8, 8, 0.08) 93.75%, rgba(8, 8, 8, 0.09) 100% )',
+        '--skeletonGradientLighter':
+            'linear-gradient( 90deg, rgba(8, 8, 8, 0.36) 0%, rgba(8, 8, 8, 0.32) 6.25%, rgba(8, 8, 8, 0.20) 12.5%, rgba(8, 8, 8, 0.04) 25%, rgba(8, 8, 8, 0.20) 37.5%, rgba(8, 8, 8, 0.32) 43.75%, rgba(8, 8, 8, 0.36) 50%, rgba(8, 8, 8, 0.08) 56.25%, rgba(8, 8, 8, 0.20) 62.5%, rgba(8, 8, 8, 0.04) 75%, rgba(8, 8, 8, 0.20) 87.5%, rgba(8, 8, 8, 0.32) 93.75%, rgba(8, 8, 8, 0.36) 100% )',
+        '--plasmaColors-whitePrimary': '#F5F5F5F5',
```
Comment (Yeti-or) [inline packages/plasma-tokens/src/themes/sberonline__light.ts:121]: а почему у нас тут старые токены еще есть ?

Comment (neretin-trike) [reply packages/plasma-tokens/src/themes/sberonline__light.ts:121]: они во все новые темы заезжают для обратной совместимости на наших компонентах (вдруг кто-то захочет использовать)

**Pull request merged**